libraries/lib-datahandler/src/main/thrift/projects.thrift

```diff
@@ -156,6 +160,13 @@ struct ProjectWithReleaseRelationTuple {
     2: required ProjectReleaseRelationship relation,
 }
 
+struct ProjectTodo {
+    1: required string todoId;
+    2: required string userId;
+    3: required string updated;
```

Comment:
The rest of the code suggests that this is a timestamp? Maybe use another data type? Some thrift definitions uses something like `i64` which seems to become a long in the java object then.

Reply:
Actually all thift definitions I found use a string, that's why I chose a string.

Reply:
Yeah, it seems we have both versions in the code base. It's fine then.